### DISCUSSION TITLE
Can we upload .h5p content

### DISCUSSION THREAD

**Discussion by dildardk123** [Lumieducation/H5P-Nodejs-library#1635] [Q&A]
Hello sir,
Can i add custom button to upload .h5p content.
I have to download content using download button via startPageRenderer.ts  of your h5p-examples.
I want to add upload button next to download to upload that .h5p downloaded content.
How can i do this?

Any suggestion will be helpful for me.

Thank you.

**Comment by sr258** [comment]
I've answered the question here: https://github.com/Lumieducation/H5P-Nodejs-library/discussions/1632#discussioncomment-1119595
